source/Jobbr.Server.UnitTests/Storage/JobbrRepositoryTests.cs

```diff
@@ -0,0 +1,232 @@
+﻿using System;
+using System.Linq;
+using Jobbr.ComponentModel.JobStorage;
+using Jobbr.ComponentModel.JobStorage.Model;
+using Jobbr.Server.Storage;
+using Microsoft.Extensions.Logging.Abstractions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+using Shouldly;
+
+namespace Jobbr.Server.UnitTests.Storage
+{
+    [TestClass]
+    public class JobbrRepositoryTests
+    {
+        private readonly IJobbrRepository _repository;
+        private readonly Mock<IJobStorageProvider> _storageProviderMock;
+
+        public JobbrRepositoryTests()
+        {
+            _storageProviderMock = new Mock<IJobStorageProvider>();
+
+            _repository = new JobbrRepository(NullLoggerFactory.Instance, _storageProviderMock.Object);
```

Comment:
Ohh `NullLoggerFactory.Instance`, I didn't know this existed. Maybe we can replace some `new NullLogerFactory()`  calls in other tests later on